src/core/solutions/brown_function.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/* 
+ * File:   brown_function.cpp
+ * Author: Alejandro Ruiz Madruga <amadruga at estudiantes.uci.cu>
+ *
+ * Created on 28 de septiembre de 2019, 21:56
+ */
+
+#include <math.h>
+#include <core/solutions/brown_function.h>
+
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace solutions
+{
+    brown_function* brown_function::make(generator* generator, unsigned int size)
+    {
+        auto* result = new brown_function(generator, size);
+        result->init();
+        return result;
+    }
+    float brown_function::calculate_fitness()
+    {
+        float result=0;

```

Comment:
Remember the coding standards: white space between operators such as equal sign, sum, multiplication, etc. there are several other issues like this. Please review.

Reply:
ok